R/regularization.R

```diff
@@ -68,7 +114,8 @@ setMethod("evaluate", signature("SmoothnessN2", "TwoStageDesign"),
                   return(1 / diff(x1_bounds) *
                              stats::integrate(integrand, x1_bounds[1], x1_bounds[2], subdivisions = 1000)$value)
```

Comment:
why do we need a maximum number of subdivisions here?

Reply:
I think we don't and removed it.

https://github.com/kkmann/otsd/blob/05a05ebe3e04965acf67afae79621af6c2b4e917/R/regularization.R#L129